html/index.html

```diff
@@ -76,6 +76,289 @@ <h1>
               </ul>
             </div>
       </section>
+
+      <section class="works-container">
+          <div class="blog-background-Image">
+          </div>
+          <div class="tonic">
+              <div>
+                  <h1>Tonic</h1>
+              </div>
+              <div class="tonic-header">
+                <ul class="canopy-list">
+                    <li class="canopies">
+                        <p >Canopy</p>
+                    </li>
+                    <li class="back">
+                        <p >Back End Dev</p>
+                    </li>
+
+                    <li class="back">
+                        <p>2015</p>
+                    </li>
+                </ul>
+
+            </div>
+
+              <div class="tonic-body">
+                  <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                  </p>
+              </div>
+
+          </div>
+          <div class="work-category">
+            <ul class="work-category-language">
+                <li>
+                    html
+                </li>
+                <li>
+                    css
+                </li>
+                <li>
+                    javascript
+                </li>
+            </ul>
+        </div>
+        <div class="Projects">
+            <button type="button">
+                See Project
+            </button>
+        </div>
```

Comment:
Please consider sticking to the Figma template of your choosing. Kindly make relevant changes.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/170815734-92829313-bafd-4dcd-a5e6-8ea5d23e2f1c.png) | ![image](https://user-images.githubusercontent.com/33268927/170815815-18d65e85-1a3f-4af5-b776-a173d9e5c4c8.png) |